SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -113,63 +128,68 @@ etc...
 
     > These lines ensure that the SpatialOS Unreal GDK can override Unreal's network replication.
```

Comment:
I think we need background info under the header about the huge new concepts we are casually mentioning in a couple of sentences in step 5. These are:
*  "check out" 
*  "workers" 
*  "schema"
*  "streaming queries"
* "Singleton Actor"

Could we add them to the GDK glossary (with a link through to SPOS docs) and pre-warn users they are going to come up.
Suggestion below.

-----
## Add GDK configurations
The steps below reference the following SpatialOS concepts: </br>
[workers] ({{URLRoot}}/content/glossary#workers) [schema]({{URLRoot}}/content/glossary#schema) [checking out]({{URLRoot}}/content/glossary#check-out] [streaming queries]({{URLRoot}}/content/glossary#streaming-queries) [{{URLRoot}}/content/glossary#singleton-actor]
You don't need to know about them in detail to complete the port.

Reply:
Sound idea. Added this.